apps/backoffice-v2/src/common/components/atoms/Home/Home.tsx

```diff
@@ -21,7 +35,21 @@ export const Home: FunctionComponent = () => {
           {firstName && ` ${firstName}`}
         </h3>
       </div>
-      <Outlet />
+      <div>
+        <Tabs defaultValue={value}>
+          <TabsList>
+            <TabsTrigger asChild={true} value="statistics">
+              <NavLink to={`/${locale}/statistics`}>Statistics</NavLink>
+            </TabsTrigger>
+            <TabsTrigger asChild={true} value="workflows">
+              <NavLink to={`/${locale}/workflows`}>Workflows</NavLink>
+            </TabsTrigger>
+          </TabsList>
+          <TabsContent value={value}>
+            <Outlet />
+          </TabsContent>
+        </Tabs>
```

Comment:
What do you think?
```suggestion
        <Tabs defaultValue={value}>
          <TabsList>
            <TabsTrigger asChild={true} value={`/${locale}/statistics`}>
              <NavLink to={`/${locale}/statistics`}>Statistics</NavLink>
            </TabsTrigger>
            <TabsTrigger asChild={true} value={`/${locale}/workflows`}>
              <NavLink to={`/${locale}/workflows`}>Workflows</NavLink>
            </TabsTrigger>
          </TabsList>
          <TabsContent value={pathname}>
            <Outlet />
          </TabsContent>
        </Tabs>
```

Reply:
I agree, as it's now more explicit and easier to understand.